src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -199,12 +159,20 @@ public MappingProfiles()
             CreateMap<Update, ReleaseNoteViewModel>();
 
             CreateMap<Comment, CommentViewModel>()
-                .ForMember(
-                    dest => dest.State,
-                    m => m.MapFrom(c => c.State.GetEnumValue())
-                );
+                .ForMember(dest => dest.CreatedBy,
+                    m => m.MapFrom(comment =>
+                        comment.CreatedById == null
+                            ? new User
+                            {
+#pragma warning disable 612
```

Comment:
I disabled the warning because Comment.LegacyCreatedBy is annoted as Obsolete but it's use here is legitmate to map it to a User name if there is no CreatedById which will be the case for new comments.